test/test614.js

```diff
@@ -0,0 +1,28 @@
+if (typeof exports === 'object') {
+    var assert = require("assert");
+    var alasql = require('..');
+}
+
+var test = '614'; // insert test file number
+
+describe('Test ' + test + ' - Read data from column irrespective of case in query', function () {
+
+    before(function () {
+        alasql('create database test' + test);
+        alasql('use test' + test);
+        alasql.options.casesensitive = false;
+    });
+
+    after(function () {
+        alasql('drop database test' + test);
+        alasql.options.casesensitive = true;
+    });
+
+    it('1) Read Data and Check Value', function (done) {
```

Comment:
Please describe the test in a way that tells what it is doing. Example could be "Mixed case excel with camel case select"